docs/docker.rst

```diff
@@ -20,6 +20,25 @@ Before building, you can modify ``Dockerfile`` to your liking, e.g. select a dif
     $ docker build -t horovod:latest horovod-docker
 
 
+Adding Mellanox RDMA support
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+If you have Mellanox NICs, it is recommended to add Mellanox OFED (MOFED) drivers in the container for best performance. Add the following to your ``Dockerfile``:
+
+.. code-block:: bash
+
+    RUN apt-get update && apt-get install -y lsb-core
+    
+    RUN wget http://www.mellanox.com/downloads/ofed/MLNX_OFED-4.4-2.0.7.0/MLNX_OFED_LINUX-4.4-2.0.7.0-ubuntu16.04-x86_64.tgz && \
```

Comment:
@alsrgv , instead of hard coded MOFED version 4.4, you should pass env variable from outside as optional field, if it is not given, you can pick default MOFED 4.4.